.github/scripts/download_assets.sh

```diff
@@ -49,9 +49,8 @@ if [ $# -gt 0 ]; then
         shift 2
     done
 else
-    download xaynia v0002
-    download smbert v0003
-    download smbert_mocked v0003
-    download sjbert v0003
-    download smroberta_tokenizer v0000
+    download qasmbert v0002
+    download smbert v0004
+    download smbert_mocked v0004
+    download e5_mocked v0000
```

Comment:
❓ it looks like this contains the regular e5 model, is that correct?

the mocked smbert model is a remnant from the times where we developed for mobile targets where we needed fast dummy models for the tests. the mocked smbert just takes the same inputs as the original smbert, then does nothing and returns a constant output in the same format as the original one. we should probably clean this up a bit once we have time.

if we can take the original e5 model for tests anyways, then there is no need to create a mocked e5 model. we should also not name the real model as mocked.

Reply:
This is a "mocked" model in that it is a tiny version of the model that otherwise matches the architecture and tokenizer. The idea was to reduce download size and increase speed of tests (it has 2 layers and embedding size of 96 I believe). If that is not a concern, I could put in the real model

Reply:
i see, if we already have it, we can stay with the mocked model for tests. faster tests are always nice to have.